code/addons/interactions/template/stories/basics.stories.ts

```diff
@@ -101,15 +102,14 @@ export const UserEventSetup = {
     const { args, canvasElement, step } = context;
     const user = userEvent.setup();
     const canvas = within(canvasElement);
-    await step('Select, type and paste on input using user-event v14 setup', async () => {
-      const input = await canvas.getByRole('textbox');
+    await step('Select and type on input using user-event v14 setup', async () => {
+      const input = canvas.getByRole('textbox');
       await user.click(input);
-      await user.type(input, 'Pasting: ');
-      await user.paste('foobar');
```

Comment:
why was this removed?

Reply:
It seems like the paste method is only working with HTTPS in the new version of user event.
It didn't seem like the paste was the point of this test, so I removed it.